include/sustainml/Engine.h

```diff
@@ -137,14 +117,29 @@ public slots:
     //! Set to true if the engine is being enabled
     bool enabled_;
 
+private slots:
+
+    void user_input_response(
+            QNetworkReply* reply);
+
+    void node_response(
+            QNetworkReply* reply);
+
 private:
 
+    const QString server_url_ = "http://127.0.0.1:5001";
+
     QString get_name_from_node_id(
             const sustainml::NodeID& id);
 
-    QString get_task_from_data(
-            const sustainml::NodeID& id,
-            void* data);
+    sustainml::NodeID get_node_id_from_name(
```

Comment:
We would need to add a brief doxygen in the new methods

Reply:
I've done it in a [follow up PR](https://github.com/eProsima/SustainML-Framework/pull/18) ([this commit](https://github.com/eProsima/SustainML-Framework/pull/18/commits/46841c1b64ac1424ddf2d65af0e073bad80ba0ee)), as well as moving them to an Utils class.